borrowings/serializers.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework import serializers
+
+from borrowings.models import Borrowing
+
+
+class BorrowingSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Borrowing
+        fields = ("id", "borrow_date", "expected_return_date", "actual_return_date", "book", "user")
+
+
+class BorrowingListSerializer(serializers.ModelSerializer):
+    book = serializers.SlugRelatedField(many=False, read_only=True, slug_field="title")
+    author = serializers.CharField(source="book.author", read_only=True)
+
+    class Meta:
+        model = Borrowing
+        fields = ("id", "book", "author")
+
+
+class BorrowingDetailSerializer(serializers.ModelSerializer):
+    book = serializers.SlugRelatedField(many=False, read_only=True, slug_field="title")
+    user = serializers.SlugRelatedField(many=False, read_only=True, slug_field="email")
+
+    class Meta:
+        model = Borrowing
+        fields = ("id", "book", "user", "borrow_date", "expected_return_date", "actual_return_date")
```

Comment:
Add user id to borrowing list serializer and detail user information to borrowing detail serializer